concepts/05 Widgets/FileManager/10 Bind to File Systems/10 Object File System.md

```diff
@@ -138,6 +327,216 @@ If the data source's field names differ from the standard field names mentioned
         //...
     ];
 
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-file-manager id="fileManager"
+        [fileSystemProvider]="objectFileProvider">
+    </dx-file-manager>  
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';    
+
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })
+
+    export class AppComponent {
+        fileItems: object[];
+        objectFileProvider: ObjectFileSystemProvider;
+
+        constructor() {
+            this.fileItems = [{
```

Comment:
```suggestion
            const fileItems = [{
```

Reply:
The fileItems is declared above the constructor, so it correct to use it in such way
export class AppComponent {
        fileItems: object[];
        ...
}

Explain why you change this?

Reply:
I didn't notice the declaration in the class. Ignore the suggestion.